packages/js/src/notifications/notifications.ts

```diff
@@ -59,11 +59,16 @@ export class Notifications extends BaseModule {
     this.#useCache = useCache;
   }
 
-  async list({ limit = 10, ...restOptions }: ListNotificationsArgs = {}): Result<ListNotificationsResponse> {
+  async list({
+    limit = 10,
+    useCache = true,
+    ...restOptions
+  }: ListNotificationsArgs = {}): Result<ListNotificationsResponse> {
     return this.callWithSession(async () => {
       const args = { limit, ...restOptions };
       try {
-        let data: ListNotificationsResponse | undefined = this.#useCache ? this.cache.getAll(args) : undefined;
+        let data: ListNotificationsResponse | undefined =
+          this.#useCache || useCache ? this.cache.getAll(args) : undefined;
```

Comment:
@SokratisVidros, if I have set `useCache: true` in the options to the `Novu` class, then this function option `useCache` won't help with the override because the first part of `or` will be satisfied. Only when the options on the `Novu` class are `use cache: false` it will control it.

Reply:
🤦 good catch.It should be the other way around.